mdc_weight_mgmt_xml_import/models/res_config_settings.py

```diff
@@ -0,0 +1,22 @@
+# © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+from odoo import models, fields
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = 'res.config.settings'
+
+    partner_notify_error = fields.Many2many(
+        related='company_id.partner_notify_error',
+        string="Notify on Error",
+        readonly=False
+    )
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+
+    partner_notify_error = fields.Many2many(
```

Comment:
Add `mdc` prefix and set a better name, according to `Many2many` field usual definition:
```suggestion
    mdc_notify_error_partner_ids = fields.Many2many(
```